pkgs/development/tools/rust/cargo-lambda/default.nix

```diff
@@ -4,22 +4,27 @@
 , makeWrapper
 , stdenv
 , Security
-, cargo-watch
 , zig
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-lambda";
-  version = "0.13.0";
+  version = "0.18.1";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-JwrTwCn4qXcjXjy4cPYM3WCblJS0J5Xxnc6WESMwCpU=";
+    sha256 = "sha256-un+GQflxhMHCMH5UEeUVsYx59ryn7MR4ApooeOuhccc=";
   };
 
-  cargoSha256 = "sha256-p3q5S6IFQQgNp/MHGSUE1DVLFyMLWDTv/dxrUACKSWo=";
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```

Comment:
This is IFD (import from derivation), which is not allowed in Nixpkgs.  You need to commit the lockfile in the package directory.